registry/registry_test.go

```diff
@@ -687,3 +736,13 @@ var _ = Describe("RouteRegistry", func() {
 		Expect(string(marshalled)).To(Equal(`{}`))
 	})
 })
+
+func portListening(addr string) bool {
```

Comment:
Hey @crhino, good catch... it was a test helper function, I must have refactored the test and forgot to dd it.  I can nix it.